.circleci/config.yml

```diff
@@ -26,16 +26,38 @@ jobs:
           command: echo "export PKG=$(echo *.tar.gz)" >> $BASH_ENV
       - run:
           name: "check package"
-          command: R CMD check --no-manual --no-build-vignettes ${PKG}
+          command: |
+            R CMD check --no-manual ${PKG}
+            result=$(tail -1 rtables.Rcheck/00check.log)
+            [[ $result = Status:* ]] || echo "could not get Status line"; set -o errexit
+            [[ $result = *"Error"* ]] && echo "checks result in errors"; set -o errexit
       - run:
           name: "install package"
           command: R CMD INSTALL ${PKG}
-#      - store_artifacts:
-#          path: rtables.Rcheck
-#          destination: rtables.Rcheck
+      - store_artifacts:
+          path: rtables.Rcheck/00check.log
+          destination: 00check.log
       - store_test_results:
           path: rtables.Rcheck/tests
 
+  roxygen:
```

Comment:
@waddella this doesn't appear to work, going into circleci I see:

```
> roxygen2::roxygenize('.', roclets = c('rd', 'collate', 'namespace'))
Warning: Version of roxygen2 last used with this package is 7.1.0.  You only have version 7.0.2
Writing NAMESPACE
Loading rtables
Loading required package: magrittr
Creating a generic function for ‘nrow’ from package ‘base’ in package ‘rtables’
Creating a generic function for ‘ncol’ from package ‘base’ in package ‘rtables’
Creating a generic function for ‘rbind’ from package ‘base’ in package ‘rtables’
Writing DM.Rd
Writing NAMESPACE
> 
> 
manuals are upto date with roxygen comments
```

The "Writing DM.Rd" suggests that the rd files were not up to date and thus this checek should have failed, correct?